### PR TITLE
install react hook form

### PR DESCRIPTION
**Pull request by russfraze**
﻿## Description of changes 
I ran yarn add react-hook-form in the frontend package folder. React hook form 7.51.1 now appears as a dependency in frontend package.json. 

## Test steps

import some basic code from the get started page into app.js and see that global functions are in scope. 

import { useForm } from "react-hook-form"

 const {
    register,
    handleSubmit,
    watch,
    formState: { errors },
  } = useForm()

https://react-hook-form.com/get-started#Quickstart

### CODE REVIEW

**Comment by perennialAutodidact** [comment]
Was RHF installed from the root directory or within `/packages/frontend`? I think it should be the latter, since it won't be used in the backend code at all.

**Comment by russfraze** [comment]
@perennialAutodidact its installed in the frontend folder, not the root.

File: package.json [hunk 1]
```diff
@@ -15,7 +15,9 @@
     "test": "npx nx run-many --target=test",
     "prepare": "husky install && chmod ug+x .husky/*"
   },
-  "dependencies": {},
+  "dependencies": {
+    "react-hook-form": "^7.51.1"
+  },
```
Comment (perennialAutodidact) [inline package.json:20]: If RHF was installed in `packages/frontend`, I don't think it'd show up in the root level `package.json`

**Review by perennialAutodidact — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (russfraze) [reply package.json]: It was actually installed in both. My bad. I removed the installation from the root, so it should be good to go.

**Pull request merged**